app/src/main/res/values-el/help.xml

```diff
@@ -174,156 +174,4 @@
         <p>Μόλις η αντίστροφη μέτρηση φθάσει στο μηδέν, χρειάζεται να ξεκινήσεις πάλι την συναλλαγή στο Exolix πηγαίνοντας
         πίσω στο προηγούμενο βήμα και μετά επιστρέφοντας στην οθόνη \"Επιβεβαίωση\".</p>
     ]]></string>
-
-    <string name="help_create_ledger"><![CDATA[
-        <h1>Create Wallet - Ledger</h1>
-        <p>You want to recover your wallet from your Ledger Nano S device.</p>
-        <p>Your secret keys never leave the Ledger device, so you need it plugged in every
-        time you want to access your wallet.</p>
-        <p>Enter a unique wallet name and password. The password is used for securing your wallet data on the Android
-        device. Use a strong password - even better use a passphrase.</p>
-        <p>Enter the block number of the first transaction used for this address in the
-        field \"Restore Height\". You can also use a date in the format YYYY-MM-DD. If you are not sure,
-        enter an approximate date/blockheight <em>before</em> you first used this wallet address.</p>
-    ]]></string>
-
-    <string name="help_wallet"><![CDATA[
-        <h1>Το Πορτοφόλι</h1>
-        <h2>Street Mode</h2>
-        <p>Street mode can be enabled/disabled in the menu or Gunther\'s head icon. In this mode, your
-        balance is not shown on any screen so you can safely use your wallet on the street, a pub or
-        other public place. Previous transactions are also hidden. New transactions will be shown, so
-        you can see that you have sent/received sweet Moneroj!</p>
```

Comment:
I already said this in the commit, but to highlight, the reason why I'm deleting translations with translations here, is that they have big english sentences that aren't just because they haven't been translated, but changed over time.
Unfortunately while I'd be fine with recycling these, there's no way to do so in a way that ensures they're not *skipped over* in the translation process. In fact, there were some examples of this happening already, with people messing with the files directly. In Weblate, this wouldn't work.
It isn't that much work, it does take a while, to translate these, but in the end it's a better idea to do it like this.